README.md

```diff
@@ -1,5 +1,254 @@
+        + [Dependency verification](#dependency-verification)
+    * [Contributors 👥](#contributors-)
+        + [Maintainers](#maintainers)
+
+<small><i><a href='http://ecotrust-canada.github.io/markdown-toc/'>Table of contents generated with
+markdown-toc</a></i></small>
+
+---
+
+## Api Documentation 📖
+
+See
+the [OpenAPI 3 here.](https://editor.swagger.io/?url=https://raw.githubusercontent.com/pagopa/pagopa-ecommerce-helpdesk-service/main/api-spec/openapi.yaml)
+
+---
+
+## Technology Stack
+
+- Kotlin
+- Spring Boot
+
+---
+
+## Start Project Locally 🚀
+
+### Prerequisites
+
+- docker
+
+### Populate the environment
+
+The microservice needs a valid `.env` file in order to be run.
+
+If you want to start the application without too much hassle, you can just copy `.env.example` with
+
+```shell
+$ cp .env.example .env
+```
+
+to get a good default configuration.
+
+If you want to customize the application environment, reference this table:
+
+| Variable name                  | Description                                                       | type              | default |
+|--------------------------------|-------------------------------------------------------------------|-------------------|---------|
+| MONGO_HOST                     | Host where MongoDB instance used to persist wallet data           | hostname (string) |         |
+| MONGO_PORT                     | Port where MongoDB is bound to in MongoDB host                    | number            |         |
+| MONGO_USERNAME                 | MongoDB username used to connect to the database                  | string            |         |
+| MONGO_PASSWORD                 | MongoDB password used to connect to the database                  | string            |         |
+| MONGO_SSL_ENABLED              | Whether SSL is enabled while connecting to MongoDB                | string            |         |
+
+### Run docker container
+
+```shell
+$ docker compose up --build
+```
+
+---
+
+## Develop Locally 💻
+
+### Prerequisites
+
+- git
+- gradle
+- jdk-17
+- ecommerce-commons library installed into maven local folder
+
+### Run the project
+
+```shell
+$ ./gradlew bootRun
+```
+
+### Install ecommerce commons library locally
+
+Into gradle build file have been added a task that take cares for you of proper ecommerce-commons library build,
+performing ecommerce-commons repository clone, checkout to the referred version and build (using maven).
+
+If you want to re-build ecommerce commons library you can run build command with the `-PbuildCommons` argument
+
+This two properties maps ecommerce commons version and ref branch:
+
+````
+val ecommerceCommonsVersion = "x.y.z" -> valued with ecommerce commons wanted pom version
+val ecommerceCommonsRef = ecommerceCommonsVersion -> the branch/tag to be checkout.
+````
+
+`ecommerceCommonsRef` is, by default, valued with the ecommerce commons library version, causing tagged version to be
+checkout. 
```

Comment:
```suggestion
`ecommerceCommonsRef` has by default the same value as `ecommerceCommonsVersion`, so that version tagged with `"x.y.z"` will be checked out and installed locally.  
```